scripts/bert/conversion_tools/convert_paddle_to_gluon.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+# encoding: utf-8
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import print_function
+
+import collections
+import os
+import sys
+import numpy as np
+import argparse
+import logging
+import json
+import mxnet as mx
+import gluonnlp as nlp
+import paddle.fluid as fluid
+
+from gluonnlp.model import BERTEncoder, BERTModel
+from gluonnlp.model.bert import bert_hparams
+from utils import get_hash, tf_vocab_to_gluon_vocab, load_text_vocab
+
+
+BAIDU_LARK_PATH = '../../../../LARK'
+if not os.path.exists(BAIDU_LARK_PATH):
+    os.system('git clone https://github.com/PaddlePaddle/LARK.git ' + BAIDU_LARK_PATH)
```

Comment:
Please make `BAIDU_LARK_PATH` a command line argument or change it to the current folder. To keep consistency with the other conversion scripts, maybe don't automatically clone the git repo?

Reply:
OK. It's done.